fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/navbar/NavigationBar.kt

```diff
@@ -0,0 +1,110 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.toolbar.navbar
```

Comment:
I am assuming that this is a file move? or are there changes in NavigationBar.kt at the same time as the move?

Reply:
now that I think of it, it made reviewing much harder :*) 

it's both a move and a change, sorry about that.

Reply:
I see, can we separate the move? Maybe in a follow up or before this review?